appveyor.yml

```diff
@@ -7,22 +7,37 @@ branches:
 environment:
   matrix:
     - PYTHON_VERSION: 2.7
-      MINICONDA: C:\Miniconda-x64
-      WHEEL: pynwb-0.1-py2-none-any.whl
+      PYTHON: C:\Miniconda-x64
+      EXPECTED_WHEEL: pynwb-0.1-py2-none-any.whl
```

Comment:
We will have to revisit this after integrating https://github.com/NeurodataWithoutBorders/pynwb/issues/137 in the mean time, LGTM